plugins/generator-1.16.4/forge-1.16.4/templates/plant.java.ftl

```diff
@@ -438,13 +438,12 @@ import net.minecraft.block.material.Material;
 				BlockState blockstate = worldIn.getBlockState(blockpos);
 				Block block = blockstate.getBlock();
 				<#if hasCondition(data.placingCondition)>
-				boolean additionalCondition = true;
 				if (worldIn instanceof IWorld) {
 					IWorld world = (IWorld) worldIn;
 					int x = pos.getX();
 					int y = pos.getY();
 					int z = pos.getZ();
-					additionalCondition = <@procedureOBJToConditionCode data.placingCondition/>;
+					boolean additionalCondition = <@procedureOBJToConditionCode data.placingCondition/>;

```

Comment:
in 1.16 you need to declare the additional condition outside the if, like it was before the last commit

Reply:
Oh...